src/common/font_file.cpp

```diff
@@ -0,0 +1,175 @@
+/*
+ * This file is part of the Colobot: Gold Edition source code
+ * Copyright (C) 2001-2016, Daniel Roux, EPSITEC SA & TerranovaTeam
+ * http://epsitec.ch; http://colobot.info; http://github.com/colobot
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://gnu.org/licenses
+ */
+ 
+#include "common/font_file.h"
+
+#include "common/logger.h"
+#include "common/make_unique.h"
+
+#include "common/resources/inputstream.h"
+#include "common/resources/outputstream.h"
+
+#include "common/system/system.h"
+
+#include <memory>
+#include <utility>
+#include <cstring>
+#include <boost/property_tree/ini_parser.hpp>
+#include <boost/regex.hpp>
+
+namespace bp = boost::property_tree;
+
+CFontConfigFile::CFontConfigFile()
+   : m_needsSave(false)
+   , m_useCurrentDirectory(false)
```

Comment:
useCurrentDirectory was only used for unit tests, so either add unit tests or remove it from here